pkg/repositories/gormimpl/project_repo.go

```diff
@@ -50,7 +51,7 @@ func (r *ProjectRepo) Get(ctx context.Context, projectID string) (models.Project
 
 func (r *ProjectRepo) ListAll(ctx context.Context, sortParameter common.SortParameter) ([]models.Project, error) {
 	var projects []models.Project
-	var tx = r.db
+	var tx = r.db.Where("state != ?", int32(admin.Project_ACTIVE))
```

Comment:
do you mean the inverse of this condition?

Reply:
:facepalm: Good catch. Copy pasta at its peak, I even "fixed" test case to make it work.